frontend/catalyst/ag_primitives.py

```diff
@@ -349,28 +389,32 @@ def _call_python_while(loop_test, loop_body, get_state, _set_state, _nonlocals,
     return get_state()
 
 
-def while_stmt(loop_test, loop_body, get_state, set_state, nonlocals, symbol_names):
+def while_stmt(loop_test, loop_body, get_state, set_state, symbol_names, _opts):
```

Comment:
Is the `_opts` used in the function? Not sure to understand what it is.

Reply:
This is all defined in the [autograph docs](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/autograph/g3doc/reference/operators.md). We don't use it (hence the `_opts`).